src/pages/SharePage/index.tsx

```diff
@@ -9,8 +10,18 @@ import {
 } from "../../components";
 import "./SharePage.css";
 
+interface FolderInfos {
+  owner: {
+    profileImageSource: string;
+    name: string;
+  };
```

Comment:
```suggestion
type Owner = {
  profileImageSource: string;
  name: string;
}
  owner: owner: Owner
```

인라인으로 중첩된 객체의 타입을 쓰기보다 타입을 직접 지정해주는게 더 좋은 방법이라고 생각합니다